src/app/components/ATIAnalytics/params/index.ts

```diff
@@ -204,27 +212,48 @@ const createBuilderFactory = (
 };
 
 export const buildATIUrl = (
-  data: PageData,
   requestContext: RequestContextProps,
   serviceContext: ServiceConfig,
+  data?: PageData,
+  atiData?: ATIData,
 ) => {
```

Comment:
Can we change the signature of this function, so that we pass in the params by name - the preference is that if there are more than 2 params to a function to use pass by reference, rather than pass by value.

We'll also need to create a new interface for the props

```suggestion
export const buildATIUrl = ({
  requestContext,
  serviceContext,
  data,
  atiData
}: ATIUrlProps)
```

Reply:
Resolved! - `buildATIUrl` is updated to receive an object rather than individually in its signature